app/assets/javascripts/controllers/cloud_subnet/cloud_subnet_form_controller.js

```diff
@@ -62,9 +54,9 @@ ManageIQ.angular.app.controller('cloudSubnetFormController', ['$http', '$scope',
   $scope.filterNetworkManagerChanged = function(id) {
     miqService.sparkleOn();
 
-    $http.get('/cloud_subnet/cloud_subnet_networks_by_ems/' + id)
-      .then(getCloudSubnetNetworksByEmsComplete)
-      .catch(miqService.handleFailure);
+    API.get("/api/cloud_networks?expand=resources&attributes=name,id&filter[]=ems_id=" + id).then(function(data) {
+      $scope.available_networks = data.resources;
+    }).catch(miqService.handleFailure);
 
     $http.get('/cloud_subnet/cloud_tenants_by_ems/' + id)
```

Comment:
Never mind, I just read your PR description. Thanks.

`The $scope.available_tenants cannot be populated by API at the moment until there is way to retrieve the parent ems of the current ems via API.`